docs/devel/developer-guides/vagrant.md

```diff
@@ -245,7 +245,7 @@ my-nginx-kqdjk    1/1       Waiting   0          33s
 my-nginx-nyj3x    1/1       Waiting   0          33s
 ```
 
-You need to wait for the provisioning to complete, you can monitor the minions by doing:
+You need to wait for the provisioning to complete, you can monitor the nodes by doing:
```

Comment:
Since the command to run below is `salt ...`, I believe this also means monitor the salt minions. If you want to change to explicit "salt minions" for clarity, that would be reasonable.